CMakeLists.txt

```diff
@@ -507,6 +615,25 @@ if(JASPER_SUPPORT_CAN_BE_COMPILED)
     target_link_libraries(raw PUBLIC ${JASPER_LIBRARIES})
 endif()
 
+if (DNG_SDK_SUPPORT_CAN_BE_COMPILED)
+    target_link_libraries(
+        raw
+        PUBLIC
+        #     zlib
+        ${XMP_LIB_DIR}/staticXMPCore.ar
```

Comment:
Since XMP is pulled in via `add_subdirectory`, doesn't it have CMake targets that could be referenced here?

Reply:
> Since XMP is pulled in via `add_subdirectory`, doesn't it have CMake targets that could be referenced here?

It does but what seems to be happening is in the Adobe build script it moves the `.a` static libraries to being a `.ac` file in a way that Cmake isn't aware of and if you attempt to link it using the targets then it will try to link to the `.a` files which no longer exist.

I'm not sure really what the solution was other than to point to it directly like this.